app/src/pages/inside/filtersPage/filtersPage.scss

```diff
@@ -0,0 +1,47 @@
+.filter-not-found {
+  width: 100%;
+  padding: 26px 10px 20px;
+  text-align: center;
+  border-bottom: 1px solid $COLOR--gray-91;
+  box-sizing: border-box;
+}
+.filter-not-found-icon {
+  display: inline-block;
+  width: 20px;
+  margin-right: 10px;
+
+  svg {
+    position: relative;
+    top: 3px;
+
+    path {
+      fill: $COLOR--gray-80;
+    }
+  }
+}
+
+.filter-not-found-text {
+  display: flex;
+  margin: 0 0 10px;
+  color: $COLOR--gray-47;
+  align-items: center;
+  justify-content: center;
+  font-family: $FONT-SEMIBOLD;
+  font-size: 16px;
+  font-weight: 600;
+  line-height: 22px;
+
+  span {
+    max-width: 200px;
+    font-family: $FONT-REGULAR;
+    font-weight: bold;
```

Comment:
@DiscoElevator, as I remember we shouldn't use font-weight because we have special bold font for that?